GardenApp/app/src/main/res/layout/activity_view_species_info.xml

```diff
@@ -0,0 +1,81 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We probably need a major change to the way text is displayed. Each species information page should appear in the same font size to the user. Scaling the information to fit on the device will cause poor user experience if a plant's description is large on one species and short on another. The ideal format would be scaled by the width of the device, but be scrollable in it's length.